publishing/datasetAPI/put_version_test.go

```diff
@@ -20,21 +21,35 @@ import (
 
 func TestSuccessfullyUpdateVersion(t *testing.T) {
 
-	datasetID := uuid.NewV4().String()
-	editionID := uuid.NewV4().String()
-	instanceID := uuid.NewV4().String()
-
 	datasetAPI := httpexpect.New(t, cfg.DatasetAPIURL)
 
+	neo4JStore, err := neo4j.NewDatastore(cfg.Neo4jAddr, "", neo4j.GenericHierarchyCPIHTestData)
+	if err != nil {
+		log.ErrorC("unable to connect to neo4j", err, nil)
```

Comment:
Again think about using `t.Errorf(...)` instead of logging error

Reply:
Done 👍